lib/ActivityStreams.js

```diff
@@ -310,13 +317,6 @@ ActivityStreams.prototype = {
     this.send(am.actions.Response("EXPERIMENTS_RESPONSE", this._experimentProvider.data), worker);
   },
 
-  _respondToPrefsRequest() {
```

Comment:
Also the old behavior was to respond with a PREFS_RESPONSE to recommendation toggles. Is that case still covered?